tools/clang/unittests/HLSLExec/HlslExecTestUtils.cpp

```diff
@@ -53,6 +57,8 @@ static std::wstring computeSDKFullPath(std::wstring SDKPath) {
 }
 
 static UINT getD3D12SDKVersion(std::wstring SDKPath) {
+  using namespace hlsl_test;
```

Comment:
There are 9 instances of `using namespace hlsl_test;` in functions in this file. Move it to the top?